openapi/components/schemas/news/topic.yaml

```diff
@@ -2,8 +2,6 @@ type: object
 description: |
   A topic which may be applicable to a news article.
 properties:
-  id:
```

Comment:
This change means that the documented API for `/news/topics` is no longer documented correctly.

This endpoint (e.g. https://vagrant.dev.schoolbox.cloud/news/topics) returns the `id`, `name` and `sequence` fields (but not `slug` for some reason).

Reply:
I guess there was no usage of the topic slug in the ui when mr matt made it.

```
    /**
     * @return array
     */
    public function getDefaultJsonMap(): array
    {
        return [
            'id',
            'name',
            'sequence',
        ];
    }
```

Reply:
Thanks, fixed.

Reply:
Let's not try to solve too many problems at once here.

For now it's fine that `GET /news/lists/feed` returns a sequence number as well: it doesn't conflict with the documentation either, as objects are allowed to have additional properties by default (see https://json-schema.org/understanding-json-schema/reference/object.html#additional-properties).